### DISCUSSION TITLE
Non Docker install

### DISCUSSION THREAD

**Discussion by IAmHugh001** [lardbit/nefarious#227] [Ideas]
Is there any chance of getting a  non docker version in the future and maybe being able to use other torrent clients other than Transmission? Thanks

**Comment by lardbit** [comment]
If you want a non-docker installation, you'd have to also install &
configure the required dependencies as well (redis, jackett, transmission).

If you're ok with that, then see:
https://github.com/lardbit/nefarious/blob/master/docs/DEVELOPMENT.md

I only have plans on supporting transmission currently.
